acquia_cms.profile

```diff
@@ -49,7 +49,7 @@ function acquia_cms_install_tasks_alter(array &$tasks) {
 /**
  * Method that calls another method to capture the installation start time.
  */
-function acquia_cms_pre_start_print_icon($install_state) {
+function acquia_cms_pre_start($install_state) {
```

Comment:
@panshulK -If this is install task function name and appear on install screen then it should start with `install_` prefix.
cc @vishalkhode1